dids/src/main/kotlin/ion/model/Models.kt

```diff
@@ -0,0 +1,205 @@
+package web5.dids.ion.model
+
+import com.nimbusds.jose.jwk.JWK
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonClassDiscriminator
+
+/**
+ * Represents an ION document containing public keys and services.
+ *
+ * @property publicKeys List of public keys.
+ * @property services List of services.
+ */
+@Serializable
+public data class Document(
+  val publicKeys: List<PublicKey> = emptyList(),
+  val services: List<Service> = emptyList()
+)
+
+/**
+ * Represents an ION service.
+ *
+ * @property id The service ID.
+ * @property type The service type.
+ * @property serviceEndpoint The service endpoint.
+ */
+@Serializable
+public data class Service(
+  public val id: String,
+  public val type: String,
+  public val serviceEndpoint: String
+)
+
+/**
+ * Represents a public key in the ION document.
+ *
+ * @property id The public key ID.
+ * @property type The public key type.
+ * @property publicKeyJWK The public key JWK (JSON Web Key).
+ * @property purposes List of public key purposes.
+ */
+@Serializable
+public data class PublicKey(
+  public val id: String,
+  public val type: String,
+  public val controller: String? = null,
+  @SerialName("publicKeyJwk") public val publicKeyJWK: JsonWebKey? = null,
```

Comment:
I think throughout the code we have Jwt

So yea happy to change to JWT if its the suggested way, but yea needs to be consistent (maybe in a new PR)

Reply:
Just making sure. JWT is a Json Web Token. JWK is a Json Web Key. They're different things.

Assuming you meant that we should rename the field to `jwk`? If so, this is implementing the sidetree spec, which contains the names already. It needs to be `publicKeyJwk`. So leaving as is.